src/ucs/memory/rcache.c

```diff
@@ -81,20 +82,22 @@ typedef struct {
 
 #ifdef ENABLE_STATS
 static ucs_stats_class_t ucs_rcache_stats_class = {
-    .name = "rcache",
-    .num_counters = UCS_RCACHE_STAT_LAST,
-    .counter_names = {
-        [UCS_RCACHE_GETS]               = "gets",
-        [UCS_RCACHE_HITS_FAST]          = "hits_fast",
-        [UCS_RCACHE_HITS_SLOW]          = "hits_slow",
-        [UCS_RCACHE_MISSES]             = "misses",
-        [UCS_RCACHE_MERGES]             = "regions_merged",
-        [UCS_RCACHE_UNMAPS]             = "unmap_events",
-        [UCS_RCACHE_UNMAP_INVALIDATES]  = "regions_inv_unmap",
-        [UCS_RCACHE_PUTS]               = "puts",
-        [UCS_RCACHE_REGS]               = "mem_regs",
-        [UCS_RCACHE_DEREGS]             = "mem_deregs",
-    }
+    .name          = "rcache",
+    .num_counters  = UCS_RCACHE_STAT_LAST,
+    .class_id      = 0,
+    .counter_names =
+            {
+                    [UCS_RCACHE_GETS]              = "gets",
+                    [UCS_RCACHE_HITS_FAST]         = "hits_fast",
+                    [UCS_RCACHE_HITS_SLOW]         = "hits_slow",
+                    [UCS_RCACHE_MISSES]            = "misses",
+                    [UCS_RCACHE_MERGES]            = "regions_merged",
+                    [UCS_RCACHE_UNMAPS]            = "unmap_events",
+                    [UCS_RCACHE_UNMAP_INVALIDATES] = "regions_inv_unmap",
+                    [UCS_RCACHE_PUTS]              = "puts",
+                    [UCS_RCACHE_REGS]              = "mem_regs",
+                    [UCS_RCACHE_DEREGS]            = "mem_deregs",
+            }
```

Comment:
pls revert this part to use previous indentation style (even though clang-format complains)